src/components/composer/index.js

```diff
@@ -678,6 +660,48 @@ class ComposerWithData extends Component<Props, State> {
           )}
 
           <FlexRow>
+            {!dataExists ? (
+              <LoadingSelect />
+            ) : (
+              <RequiredSelector
+                data-cy="composer-community-selector"
+                onChange={this.setActiveCommunity}
+                value={activeCommunity}
+              >
+                <option key={-1} value="">
+                  Communities
+                </option>
+                {availableCommunities.map(community => {
+                  return (
+                    <option key={community.id} value={community.id}>
+                      {community.name}
+                    </option>
+                  );
+                })}
+              </RequiredSelector>
+            )}
+            {!dataExists ? (
+              <LoadingSelect />
+            ) : (
+              <RequiredSelector
+                data-cy="composer-channel-selector"
+                onChange={this.setActiveChannel}
+                value={activeChannel}
+              >
+                <option key={-1} value="">
```

Comment:
As for I know we don't need `key` prop here. we use `key` prop only when we do `.map`. though i might be wrong

Reply:
@dev-drprasad  Oh I think you were reviewing an older version actually, I made some updates to the PR afterwards to remove redundant code. As for the key parameter, I think post render the above option also becomes a part of the rendered array of options(via . map). Since all are now siblings so key should be unique I think.